From 5147e15f402d1023a68ac028f9a3ecfd5fd42f28 Mon Sep 17 00:00:00 2001
From: Marcelo Leitner <mleitner@redhat.com>
Date: Tue, 21 Oct 2014 16:20:27 -0400
Subject: [net] netfilter: nf_tables: rename nf_table_delrule_by_chain()

Message-id: <20141021162027.22888.41299.stgit@localhost.localdomain>
Patchwork-id: 97999
O-Subject: [RHEL7 net PATCH 102/124] netfilter: nf_tables: rename nf_table_delrule_by_chain()
Bugzilla: 1148041 1155088
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

Backport of upstream commit: ce24b7217b60980ff0366dc6afbd5418db9972f2

 commit ce24b7217b60980ff0366dc6afbd5418db9972f2
 Author: Arturo Borrero <arturo.borrero.glez@gmail.com>
 Date:   Tue Sep 2 16:42:24 2014 +0200

    netfilter: nf_tables: rename nf_table_delrule_by_chain()

    For the sake of homogenize the function naming scheme, let's rename
    nf_table_delrule_by_chain() to nft_delrule_by_chain().

    Signed-off-by: Arturo Borrero Gonzalez <arturo.borrero.glez@gmail.com>
    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>

Signed-off-by: Marcelo Ricardo Leitner <mleitner@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/netfilter/nf_tables_api.c b/net/netfilter/nf_tables_api.c
index d284b1c..5328d9d 100644
--- a/net/netfilter/nf_tables_api.c
+++ b/net/netfilter/nf_tables_api.c
@@ -1902,7 +1902,7 @@ static int nft_delrule(struct nft_ctx *ctx, struct nft_rule *rule)
  return 0;
 }
 
-static int nf_table_delrule_by_chain(struct nft_ctx *ctx)
+static int nft_delrule_by_chain(struct nft_ctx *ctx)
 {
  struct nft_rule *rule;
  int err;
@@ -1955,12 +1955,12 @@ static int nf_tables_delrule(struct sock *nlsk, struct sk_buff *skb,
 
    err = nft_delrule(&ctx, rule);
   } else {
-   err = nf_table_delrule_by_chain(&ctx);
+   err = nft_delrule_by_chain(&ctx);
   }
  } else {
   list_for_each_entry(chain, &table->chains, list) {
    ctx.chain = chain;
-   err = nf_table_delrule_by_chain(&ctx);
+   err = nft_delrule_by_chain(&ctx);
    if (err < 0)
     break;
   }
-- 
1.7.1